client-gen/src/snapshots/sails_clientgen__generator__tests__basic_works.snap

```diff
@@ -6,31 +6,37 @@ expression: generate(program).unwrap()
 use parity_scale_codec::{Decode, Encode};
 #[allow(unused_imports)]
 use sails_rtl::collections::BTreeMap;
-use sails_rtl::{String, *};
-#[allow(unused_imports)]
-use sails_sender::{Call, CreateProgramCall, GStdSender};
+use sails_rtl::{
+    calls::RemotingAction, gstd::calls::Args, gstd::calls::Remoting, prelude::*, String,
+};
 pub trait Service {
```

Comment:
This has to be a generic trait, exactly like in the snippet I sent over to you the other day

Reply:
Take a look now